vcenter/vcenter.py

```diff
@@ -71,6 +71,46 @@ def search_for_snapshot(self, vm, snapshot_name):
         raise ValueError('snapshot {} cannot be found'.format(snapshot_name))
         return None
 
+    def __search_machine_by_name(self, vm_name):
+        objView = self.content.viewManager.CreateContainerView(self.content.rootFolder,
+                                                               [vim.VirtualMachine],
+                                                               True)
+        vm = None
```

Comment:
Very nice construction, however, the following one is recommended for these cases in Python:
```
vm = next(item for item in objView.view if item.name == vm_name, None)
```

It basically takes first matching item, returning None in case no matching item was found.